x-pack/plugins/ml/public/application/memory_usage/memory_usage_page.tsx

```diff
@@ -24,7 +24,7 @@ export const MemoryUsagePage: FC = () => {
   const [selectedTab, setSelectedTab] = useState<TAB>(TAB.NODES);
   useTimefilter({ timeRangeSelector: false, autoRefreshSelector: true });
 
-  const isServerless = useIsServerless();
+  const { showNodeInfo } = useEnabledFeatures();
```

Comment:
In the Observability project, the Memory Usage page is failing to load, with a `404` error:

<img width="833" alt="image" src="https://github.com/elastic/kibana/assets/7405507/6022a2c3-0d34-4044-a630-35a39179c605">

Reply:
In the Security page, there is no `404` but the response is empty:

<img width="1114" alt="image" src="https://github.com/elastic/kibana/assets/7405507/895067ee-7a96-4f64-a8a9-282b86bd27b8">

Reply:
Fixed in https://github.com/elastic/kibana/pull/166651/commits/14e1f50999ad23d9fef344c9086d059fb035e6f8
I had incorrectly moved the model management route registration behind a check for `dfa` and `nlp` features, thinking the routes are for trained model management only.
However these routes are for AD models too.